tracks/ruby/exercises/acronym/mentoring.md

```diff
@@ -22,11 +22,23 @@ with variations for the regex (see below).
 In all these cases: it's recommended to suggest the `String#scan` method and, because the exercise is not about Regular Expressions, give them also the RegEx (`/\b[a-zA-Z]/`). (At this point in the track, it's more about experiencing the strengths of Ruby's built-in methods than making them find them. There's ample opportunity for research later on in the track.)   
 
 ## Talking points
-* If the solution uses Class instead of Module: that's totally acceptable at this stage. No reason to discuss.
+* `Class vs Module`: If the solution uses Class instead of Module: that's totally acceptable at this stage. No reason to discuss.
 * Because of the regex needed for `split`, `scan` is in this exercise (marginally) faster.
 * It can be hard to catch all the possible delimiters. 
 * `upcasing` the whole string is (10% ?) slower than `upcasing` the acronym only. 
 * Store the regex in a constant or a variable, mostly so it can be named. 
+* `join("")`: The `("")` after `join` can be omitted, because it's the default for `join`.
+
+
+## Mentoring strategy
+* When they start with `split`, with or without `tr` or `gsub`, and `map` to `upcase`: start with pointing them to `scan`, name what it accomplished afterwards:
+1) (first comment)
```

Comment:
If you indent this at least four spaces, you get a "copyable" code block, see [here](https://github.com/exercism/website-copy/blob/master/tracks/javascript/exercises/leap/mentoring.md). 

```markdown
## Mentoring strategy

* When they start with `split`, with or without `tr` or `gsub`, and `map` to `upcase`: start with pointing them to `scan`, name what it accomplished afterwards:
  1. (first comment)

      Using split works, but it forces you to focus on the delimiters (space, -).
      
      Ruby has `String#scan`. With `scan` and a regex (Regular Expression), you can catch the parts you 
      need. For instance, `scan` with the regex `/\b[a-zA-Z]/` will catch 'word boundaries' (space, - and 
      more) followed by a word character.
      
      I use "www.rubular.com" whenever I want to check a regex.
  
  2. (after they implemented scan)
 
      You just discovered one of the strengths of Ruby: we can extract certain parts of a string into an 
      array in only one or two operations most of the time. :-)
      
      With `scan`, you avoid the multiple iteration. You also got rid of the intermediate variable 'xxx', that
      you needed to track stuff in your loop. That's called the Accumulator pattern. In general, eliminating 
      the extra variable is considered best practice in Ruby.
```

This might be preferable, as it's nicer than having to correctly select between the quotes (`"`)

Reply:
(If you change to that, make sure you _Preview_ it on GH 🗡 . It should appear as code block.

Reply:
I looked into this, but I'm not a fan of using non-code in code blocks. I also don't know how that will translate to screen readers. So, I'm leaving it as is for now.

Reply:
Since this is markdown, then perhaps as a "quote" instead? 

```markdown
> Using split works, but it forces... etc.
```

If we have this as a running standard, then perhaps we can pair this with functionality that allows a click to copy button for those using graphical browsers.

Reply:
Excellent!